raiden/ui/app.py

```diff
@@ -236,7 +240,8 @@ def run_app(
             config=config,
             chain=blockchain_service,
             query_start_block=BlockNumber(start_block),
-            default_one_to_n_address=one_to_n_contract_address,
+            default_one_to_n_address=one_to_n_contract_address
+            or contracts[CONTRACT_ONE_TO_N]["address"],
```

Comment:
This looks very weird, maybe adding parens would make it more readable (or does black remove those?)

Reply:
I copied the style from below, but added parens now.